java-library/build.gradle

```diff
@@ -30,6 +30,7 @@ targetCompatibility = 1.7
 dependencies {
     compile project(':kmp-library')
     implementation 'com.squareup.okio:okio:2.2.2'
+    implementation "io.ktor:ktor-client-okhttp:$ktor_version"
```

Comment:
Thought about that again and I don't think we need it. Instead, we can change the `kmp-library/build.gradle` to expose `ktor` as part of the api using `api "io.ktor:ktor-client-core-jvm:$ktor_version"` instead of `implementation  "io.ktor:ktor-client-core-jvm:$ktor_version"` in the `jvmMain` closure.

Reply:
71b727c32c4be62f7feb2589bf01eb8e60f48112